### DISCUSSION TITLE
2024 frontend development experience with Next.js

### DISCUSSION THREAD

**Discussion by babakfp** [vercel/next.js#73318] [Show and tell] [answered]
https://github.com/user-attachments/assets/d901422a-3f70-4f70-a53f-01842ecea25b

Every fucking Next.js app is like this.

**Comment by icyJoseph** [comment, marked as answer]
Yeah, that's React hydration errors because of browser extensions, that they promised would be fixed with React 19, https://react.dev/blog/2024/04/25/react-19#compatibility-with-third-party-scripts-and-extensions , but here we are - not a Next.js issue as far as I can see ~ to be fair, having extensions that also modify the HTML before scripts run is a bit iffy... 
 
Unfortunately, for the time being, you'd need to develop without dark-reader 🤷‍♂️ or... apply `suppressHydrationWarning`, to the body tag (assuming that's where dark-reader hijacks the DOM).
